src/update/UpdateManager.cpp

```diff
@@ -70,25 +69,27 @@ void UpdateManager::GetUpdates()
     auto response = audacity::network_manager::NetworkManager::GetInstance().doGet(request);
 
     response->setRequestFinishedCallback([response, this](audacity::network_manager::IResponse*) {
-
+        using namespace BasicUI;
         if (response->getError() != audacity::network_manager::NetworkError::NoError)
         {
-            wxTheApp->CallAfter([] {ShowExceptionDialog(nullptr,
+            wxTheApp->CallAfter([] {ShowErrorDialog( {},
```

Comment:
Why not BasicUI::Services::CallAfter?

Reply:
This PR may be inconsistent about whether it changes all existing uses to use BasicUI.  I do change all uses of ShowExceptionDialog.  I don't, for CallAfter.

I mean to change just enough uses of CallAfter, as needed, to make other libraries extracted later, platform-neutral by use of BasicUI.

Reply:
In fact, after rebasing to later master,  CallAfter isn't there, but CallAfterRecording instead, and that function won't be in BasicUI.